parcels/particleset.py

```diff
@@ -227,7 +235,8 @@ def _convert_to_reltime(time):
         initialised = set()
         for v in self.ptype.variables:
             if v.name in ['xi', 'yi', 'zi', 'ti']:
-                self.particle_data[v.name] = np.empty((len(lon), fieldset.gridset.size), dtype=v.dtype)
+                ngrid = fieldset.gridset.size if fieldset is not None else 1
+                self.particle_data[v.name] = np.empty((len(lon), ngrid), dtype=v.dtype)
```

Comment:
ah, okay, now I see why 'ngrid' should not be 0 ... the variable naming semantic here is a bit misleading then, but I understand the purpose.